.github/workflows/build.yml

```diff
@@ -1,4 +1,4 @@
-name: Build
+name: Build API
```

Comment:
What's wrong with "Build"? API is the only thing that is being built in this repo